testing_and_setup/compass/ocean/cosine_bell/write_graph_info.py

```diff
@@ -12,7 +12,9 @@
 from mpas_tools.conversion import convert
 from mpas_tools.io import write_netcdf
 
+
 write_netcdf(convert(xarray.open_dataset('base_mesh.nc'),
```

Comment:
Put 2 blank lines between imports and the beginning of the script or the first function definition.